v1/terraform_test.go

```diff
@@ -0,0 +1,53 @@
+package atlas
+
+import (
+	"bytes"
+	"reflect"
+	"testing"
+)
+
+func TestTerraformConfigLatest(t *testing.T) {
+	server := newTestAtlasServer(t)
+	defer server.Stop()
+
+	client, err := NewClient(server.URL.String())
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	actual, err := client.TerraformConfigLatest("hashicorp", "existing")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	expected := &TerraformConfigVersion{
+		Version:  5,
+		Metadata: map[string]string{"foo": "bar"},
+	}
+
+	if !reflect.DeepEqual(actual, expected) {
+		t.Fatalf("%#v", actual)
+	}
+}
+
+func TestCreateTerraformConfigVersion(t *testing.T) {
+	server := newTestAtlasServer(t)
+	defer server.Stop()
+
+	client, err := NewClient(server.URL.String())
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	v := &TerraformConfigVersion{
+		Version:  5,
```

Comment:
Yea - this is handled by Atlas, you shouldn't pass this in, but it will be returned to you.